tests/foreman/api/test_discoveryrule.py

```diff
@@ -150,6 +163,19 @@ def test_positive_update_name(self):
                 discovery_rule = discovery_rule.update(['name'])
                 self.assertEqual(discovery_rule.name, name)
 
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_update_by_associating_org_loc(self):
```

Comment:
test is to update org and location. so will update with: `test_positive_update_with_org_loc`